InteractiveHtmlBom/ibom.js

```diff
@@ -546,6 +558,11 @@ function setBomCheckboxes(value) {
 
 document.onkeydown = function(e) {
   switch (e.key) {
+    case "n":
+      checkBomCheckbox(currentHighlightedRowId, 2);
```

Comment:
Don't assume that "Placed" checkbox is # 2.
Find one that matches "Placed" or "placed" from the list that is stored in global `checkboxes` array